.php_cs.dist

```diff
@@ -19,7 +19,7 @@ $finder = PhpCsFixer\Finder::create()
 $config = PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
-        '@PHP56Migration' => true,
+        '@PHP56Migration:risky' => true,
```

Comment:
we don't need it here, it's part of `@PHPUnit60Migration:risky` that we are using already

Reply:
Is it? I cannot see [it](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/src/RuleSet.php#L358).

Reply:
It is.

Reply:
How?
I see `PHP56Migration:risky` is part of `@PHP70Migration:risky`.
Then `@PHP70Migration:risky` is part of `@PHP71Migration:risky`.
And that's it, how it becomes a part of `@PHPUnit60Migration:risky`?

Reply:
@keradus take a look at: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4432/commits/7b65acacc2d589fcafe48e86e6310a8491b3fd63 - I have removed it and now it's OK to have `pow` function (which changing to `**` is part of `@PHP56Migration:risky` set).

Reply:
`PHPUnit60Migration:risky` has `@PHPUnit56Migration:risky` it does indeed not have `PHP56Migration:risky` IIUC

Reply:
Aaah. Indeed, I read them both as phpunit rulesets. My bad, sorry

Reply:
no worries :)